static/js/settings_display.js

```diff
@@ -85,6 +102,13 @@ exports.set_up = function () {
         change_display_setting(data, '#display-settings-status');
     });
 
+    $('#demote_inactive_streams').change(function () {
+        var demote_inactive_streams = this.value;
+        var data = {};
+        data.demote_inactive_streams = demote_inactive_streams;
```

Comment:
The above three lines can be combinedly written as `var data = {demote_inactive_streams: this.value};` making the code more clearer.